composer.json

```diff
@@ -12,7 +12,8 @@
     "require-dev": {
         "fzaninotto/faker": "~1.4",
         "mockery/mockery": "0.9.*",
-        "phpunit/phpunit": "~5.7"
+        "phpunit/phpunit": "~5.7",
+        "anhskohbo/no-captcha": "^2.3"
```

Comment:
Don't add this to require-dev move it to require we don't need dev dependencies all the time specially in production